docs/userToken.md

```diff
@@ -1,14 +1,17 @@
 # User Token
 
 ## Static token
-
 ```js
 const client = new StreamChat('api_key');
 client.connectUser({ id: 'vishal' }, 'user_token_string');
 ```
 
 ## Token Provider
 
+You can use this feature to use short-lived token within your app. When the token expires, the WS connection will get a new token from the Token Provider and reconnect.
+
+Make sure the TokenProvider function always resolves, you can wrap your async function with an async-retryable to not reject the promise immediately in case of a network issue. 
```

Comment:
do you think providing an example for this scenario would be better ?

Reply:
yea adding